prometheus/prometheus.py

```diff
@@ -213,7 +220,7 @@ def init(options, configuration, plugin):
 
 plugin.add_option(
     'prometheus-listen',
-    '0.0.0.0:9750',
+    '0.0.0.0:9900',
```

Comment:
You should not modify this, `9750` we modified it lately as it's now The Official C-lightning Prometheus Port :)